__tests__/app.test.js

```diff
@@ -1,8 +1,8 @@
-const db  = require("../db/connection");
```

Comment:
be careful with all of these tiny formatting changes, I would recommend if you haven't installing prettier to make sure the formatting is consistent as you could end up in merge conflict land because your changing quite a few things each merge :)